### DISCUSSION TITLE
smarty plugin extension not callable

### DISCUSSION THREAD

**Discussion by Murlocman** [smarty-php/smarty#1092] [Q&A]
Hi,
i struggle to understand where to put my extension. I have put the block translate tag into the extension like here [https://smarty-php.github.io/smarty/5.x/api/extending/block-tags/]. We load smarty via composer/autoload. Shouldnt it get automaticly loaded if its inside the smarty folder?
We want to steer away form addPluginsDir().

**Comment by wisskid** [comment]
This page explains creating extensions in (hopefuly sufficient) detail: https://smarty-php.github.io/smarty/5.x/api/extending/extensions/
